### ISSUE TITLE
RoleMixin.add_permissions documentation inconsistency

### ISSUE THREAD

**Issue by WardBenjamin** [Flask-Middleware/flask-security#432]
Hi there,

The [documentation](https://flask-security-too.readthedocs.io/en/stable/api.html#flask_security.RoleMixin.add_permissions) for `add_permissions` specifies:

> Deprecated since version 3.4.4: Use UserDatastore.remove_permissions_from_role()

However, the [in-code documentation](https://github.com/Flask-Middleware/flask-security/blob/9c874d9fbff0d08f6b6af9e769a7fcec82e4dd4c/flask_security/core.py#L648) for the `RoleMixin` correctly specifies the deprecation and new interface:

```py
.. deprecated:: 4.0.0
    Use :meth:`.UserDatastore.add_permissions_to_role`
```

The documentation might just need to be rebuilt on RTD.

**Comment by jwag956** [comment]
Actually - it is incorrect in the code for 3.4.5 and has been fixed in master (which will be 4.0).

**Comment by WardBenjamin** [comment]
Ah, good catch - thanks!
